data/101/844/515/101844515.geojson

```diff
@@ -32,15 +30,16 @@
     "qs:pop":0,
     "qs:source":"TCustom EuroGlobalMap + UMZ Urban Polygons + Geonames + GeoPlanet",
     "qs:type":"rural point-near-poly",
-    "src:geom":"quattroshapes",
-    "src:geom_alt":[],
-    "src:lbl_centroid":"mapshaper",
+    "src:geom":"whosonfirst",
+    "src:geom_alt":["quattroshapes"],
+    "wof:geom_alt":["quattroshapes"],
```

Comment:
I'm n sure about this `"wof:geom_alt"` property, is it valid?

Reply:
Valid property ([info](https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/geom_alt.json)), but needed to be exportified.. updated in most recent commit.